lib/generators/templates/tokenable.rb.erb

```diff
@@ -0,0 +1,3 @@
+Tokenable::Config.lifespan = nil
```

Comment:
Should we add a sensible default for expiry? Even something like 7 days would be better than infinite

Reply:
Yeah i was going to discuss that with you sometime. It feels like it needs a default.

Reply:
Will add in another PR.